File: prob10.js
// https://projecteuler.net/problem=10
let sum = 0;

for(let currentNum = 2; currentNum < 2000000; currentNum++){
  if(isPrime(currentNum)){
    sum += currentNum
  }
}

function isPrime(currentNum){ 
  for(let divisor = 2; divisor <= Math.round(Math.sqrt(currentNum)); divisor++){
    if(currentNum % divisor === 0){
      return false
    }
  }
  return true
}

console.log(sum)

File: prob10.py
# https://projecteuler.net/problem=10
# Completed 12/31/2021

import math

def is_prime(num):
    for i in range(2, math.floor(math.sqrt(num)) + 1):
        if num % i == 0:
            return False
    return True

sum = 0
for i in range(2, 2000000):
  if is_prime(i):
    sum += i
    
print(sum)

